custom_components/tuya_local/devices/lytmi_fantasy3_hdmi_syncbox.yaml

```diff
@@ -93,6 +95,7 @@ secondary_entities:
           max: 1000
         mapping:
           - scale: 10
+            step: 1
```

Comment:
This one is not needed, the step defaults to 1.  If your intention was to make the UI step 1.0 instead of 0.1 after scaling then you would need to make the step 10.  That would actually fit with the precision of 0.

Reply:
You are correct, I fixed that locally and now it behaves as expected